draft-ietf-quic-tls.md

```diff
@@ -1171,27 +1171,34 @@ Note:
 ## Receiving Out-of-Order Protected Frames {#pre-hs-protected}
 
 Due to reordering and loss, protected packets might be received by an endpoint
-before the final TLS handshake messages are received.  A client will be unable
-to decrypt 1-RTT packets from the server, whereas a server will be able to
-decrypt 1-RTT packets from the client.
+before the final TLS handshake messages are received.  Each endpoint MUST NOT
+decrypt 1-RTT packets from its peer prior to completing the handshake.
```

Comment:
I'd leave the old text in for emphasis, and also a minor edit: "A client will be unable to decrypt 1-RTT packets from the server, whereas a server will be able to decrypt 1-RTT packets from the client. Both endpoints MUST NOT decrypt 1-RTT  packets from their peers prior to completing the handshake."

Reply:
Meh. Both/Each, they're both sucky here.